### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

File: contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml [hunk 1]
```diff
@@ -1487,7 +1487,7 @@ data:
     # to users that `kubectl edit` this config map.
     #
     # These sample configuration options may be copied out of
-    # this example block and unindented to be in the data block
+    # this example block and unintended to be in the data block
```
Comment (str4d) [inline contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml:1490]: The original spelling was intended 🥁🔔

File: contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml [hunk 2]
```diff
@@ -1760,7 +1760,7 @@ data:
...
```
Comment (str4d) [inline contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml:1763]: Ditto.

**Review by str4d — CHANGES_REQUESTED** [review]

File: contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml [hunk 3]
```diff
@@ -1501,7 +1501,7 @@ data:
     # default-managed-by-label-value contains the default value given to the
     # "app.kubernetes.io/managed-by" label applied to all Pods created for
     # TaskRuns. If a user's requested TaskRun specifies another value for this
-    # label, the user's request supercedes.
+    # label, the user's request supersedes.
```
Comment (daira) [inline contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml:1504]: "supercedes" is just the British English spelling. We don't normally worry about which variant of English is used in comments.

File: contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml [hunk 4]
```diff
@@ -1487,7 +1487,7 @@ data:
     # to users that `kubectl edit` this config map.
     #
     # These sample configuration options may be copied out of
-    # this example block and unindented to be in the data block
+    # this example block and unindended to be in the data block
```
Comment (daira) [inline contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml:1490]: This is incorrect and should be reverted.

File: contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml [hunk 5]
```diff
@@ -1760,7 +1760,7 @@ data:
...
```
Comment (daira) [inline contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml:1763]: This is incorrect and should be reverted.

File: contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml (hunk 3 above)
Comment (daira) [reply contrib/ci-builders/tekton/tekton-labs/releases/tekton-pipeline-v0.37.0.yaml:1504]: Apparently not specifically British, but a common spelling for this word nonetheless. It comes from the Middle French _superceder_, which is good enough for me.

**Review by nuttycom — APPROVED** [review]
ACK, doc/comment-only change.

**Review by str4d — APPROVED** [review]

**Pull request merged**